windows-container-samples/msbuild/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM microsoft/windowsservercore
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]
+
+RUN (New-Object System.Net.WebClient).DownloadFile('http://download.microsoft.com/download/5/f/7/5f7acaeb-8363-451f-9425-68a90f98b238/visualcppbuildtools_full.exe', 'visualcppbuildtools_full.exe') ; \
+    Start-Process .\visualcppbuildtools_full.exe -ArgumentList '/NoRestart /S' -Wait ; \
+    rm visualcppbuildtools_full.exe
+
+COPY entrypoint.bat entrypoint.bat
+
+SHELL ["cmd"]
+
+WORKDIR /code
+
+ENTRYPOINT ["C:\\entrypoint.bat"]
```

Comment:
Lost and found. Perhaps instead of using this `entrypoint.bat` we should burn in all environments as done here: https://github.com/friism/hello-world/blob/32b486472ebcd64fd26b0264dc988958bf44ccf5/windows/Dockerfile.build#L14-L19